### PR TITLE
fix: force PEER_DNS servers through the vpn tunnel

### PR DESCRIPTION
**Pull request by ferringb**
## Breaking change

```
```

## Proposed change

```txt
This PR fixes cidr conflicts between LOCAL_NETWORK and
any NS advertised by the provider; see #1667 for an example.

Specifically, PIA advertised 10.0.0.243, users have `LOCAL_NETWORK=10.0.0.0/8`.

This results in 10.0.0.243 traffic being routed around the tunnel, rather than through it.
```

## Type of change

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality to a this container)
- [ ] Breaking change (fix/feature causing existing functionality to break)

## Additional information

- This PR fixes or closes issue: ```fixes #1713, #1667 ```

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.

If user exposed functionality or configuration variables are added/changed:

- [x] Documentation added/updated

### CODE REVIEW

File: docs/config-options.md [hunk 1]
```diff
@@ -14,6 +14,8 @@
 | `OPENVPN_OPTS`      | Will be passed to OpenVPN on startup                                                                | See [OpenVPN doc](https://openvpn.net/index.php/open-source/documentation/manuals/65-openvpn-20x-manpage.html) |
 | `LOCAL_NETWORK`     | Sets the local network that should have access. Accepts comma separated list.                       | `LOCAL_NETWORK=192.168.0.0/24`                                                                                 |
 | `CREATE_TUN_DEVICE` | Creates /dev/net/tun device inside the container, mitigates the need mount the device from the host | `CREATE_TUN_DEVICE=true`                                                                                       |
+| `PEER_DNS`          | Controls whether to use the DNS provided by the OpenVPN endpoint. | To use your host DNS rather than what is provided by OpenVPN, set `PEER_DNS=no`.  This allows for potential DNS leakage. |
+| `PEER_DNS_PIN_ROUTES` | Controls whether to force traffic to peer DNS through the OpenVPN tunnel. | To disable this default, set `PEER_DNS_PIN_ROUTES=no`. |
```
Comment (pkishino) [inline docs/config-options.md:18]: Are you going to add these to default settings file as well?

**Comment by pkishino** [comment]
Looks good, just think these should be added to defaults

**Comment by ferringb** [comment]
> Looks good, just think these should be added to defaults
So uh... which defaults we talking about here? :)

If you're referring to the Dockerfile env defaults- I'll push a commit shortly addressing that.  If you're looking for something else, please clarify (I doubt you're referring to `default-settings.json` since that's transmission config).

**Comment by pkishino** [comment]
yeah sorry, I meant the dockerfile.. battling bit of a cold currently, not in the right mind :P

**Review by pkishino — APPROVED** [review]

**Comment by mustanggb** [comment]
Why is it using `yes/no` when other env flags are using `true/false`.

I think it would make more sense to users to stick with one standard.

**Comment by pkishino** [comment]
Wow, 
That’s bad of me to miss that.. I’ll see if I can fix that later tonight. You are right, we should stick to true and false

**Comment by ferringb** [comment]
See #1820 ; I cut a quick PR for changing the configurables in the way y'all are discussing above.

**Pull request merged**